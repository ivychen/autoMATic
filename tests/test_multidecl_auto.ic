/* autoMATic Compiler - Spring 2018 - COMS4115 PLT
 by Ivy Chen ic2389, Nguyen Chi Dung ncd2118,
 Nelson Gomez ng2573, Jimmy O'Donnell jo2474 */

int main() {
    auto x = 5, y = "test", z = 4.0;
    println(x);
    println(y);
    println(z);
    return 0;
}
