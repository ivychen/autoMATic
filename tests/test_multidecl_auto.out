5
test
4.000000
